packages/error-reporting/package.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
We have a few scripts defined in our root `package.json` in this repo:

```json
{
    "postinstall": "node ./scripts/install.js",
    "link-common": "node ./scripts/link-common.js",
    "update-deps": "node ./scripts/update-deps.js",
    "docs": "node ./scripts/docs",
    "prepare-ghpages": "node ./scripts/docs/prepare.js",
    "remove-ghpages": "node ./scripts/docs/remove.js",
    "lint": "jshint scripts/ packages/ system-test/ test/ && jscs packages/ system-test/ test/",
    "test": "npm run unit-test && npm run docs && npm run snippet-test",
    "unit-test": "mocha --timeout 5000 --bail packages/*/test/*.js",
    "snippet-test": "mocha --timeout 5000 --bail test/docs.js",
    "system-test": "mocha packages/*/system-test/*.js --no-timeouts --bail",
    "cover": "istanbul cover node_modules/mocha/bin/_mocha --report lcovonly -x 'packages/*/src/*{/*,/**/*}.js' -- --no-timeouts --bail packages/*/test/*.js -R spec",
    "coveralls": "npm run cover && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js"
  }
```

We will need an exclusion in the `unit-test`, `snippet-test`, `system-test`, and `cover` tasks to ignore files in the `error-reporting` directory.

Additionally, `scripts/docs/config.js` will need a block for `error-reporting` in the format of `logging-bunyan` (basically just an object with `skip: true`. This excludes the docs from being included as a member of the `google-cloud` package.

@callmehiphop 

  - can you think of other edits that need to be made to exclude error-reporting from our testing procedure
  - anything else they need to do to make sure they end up in our docs site as a module (but not as a part of the `google-cloud` package?